invoker/core/src/test/java/com/google/cloud/functions/invoker/http/HttpTest.java

```diff
@@ -365,31 +377,37 @@ private void httpResponseSetAndGet(
       HttpClient httpClient = new HttpClient();
       httpClient.start();
       String uri = "http://localhost:" + serverPort;
-      Request request = httpClient.POST(uri);
+      org.eclipse.jetty.client.Request request = httpClient.POST(uri);
       ContentResponse response = request.send();
       assertThat(response.getStatus()).isEqualTo(HttpStatus.OK_200);
       throwIfNotNull(exceptionReference.get());
     }
   }
 
-  private static class HttpResponseServlet extends HttpServlet {
+  private static class HttpResponseHandler extends Handler.Abstract {
     private final AtomicReference<HttpResponseTest> testReference;
     private final AtomicReference<Throwable> exceptionReference;
 
-    private HttpResponseServlet(
+    private HttpResponseHandler(
         AtomicReference<HttpResponseTest> testReference,
         AtomicReference<Throwable> exceptionReference) {
       this.testReference = testReference;
       this.exceptionReference = exceptionReference;
     }
 
     @Override
-    protected void doPost(HttpServletRequest req, HttpServletResponse resp) {
+    public boolean handle(Request request, Response response, Callback callback) {
+      if (!HttpMethod.POST.is(request.getMethod())) {
```

Comment:
Any reason why this handler handles non-POST requests differently than above?

Reply:
It should have been the same, but now it has been removed.